src/main/scala/nl/knaw/dans/easy/multideposit/Main.scala

```diff
@@ -167,7 +167,7 @@ object Main {
 
     actions
       .doOnNext(action => log.info(s"Checking preconditions of ${action.getClass.getSimpleName} ..."))
-      .foldLeft(ActionAndResult())(_ += _)
+      .foldLeft(ActionAndResult())((acc, cur) => acc += cur)
```

Comment:
we rewrote this when I explained this to you, but we forgot to revert it... Please put this back to `_ += _`